### ISSUE TITLE
[debops-contrib.dropbear_initramfs] authorized_keys location (buster)

### ISSUE THREAD

**Issue by evilham** [debops/debops#656]
Documenting so I can confirm + fix over the weekend.

I ran against an issue a few days ago, where the `authorized_keys` file is expected under `/etc/dropbear-initramfs` on a buster-based system.

If I confirm that, I guess the solution would be to research exactly in which package version this changed and detect that and act accordingly?

**Comment by drybjed** [comment]
Yes, going via a detected package version could work better than via an OS release. You could add the local facts for the role before the configuration happens and check the package version via the fact.

**Comment by julienmalik** [comment]
I confirm I spent a few hours figuring out what seems a similar issue.

On ubuntu xenial, the package to install is dropbear-initramfs so the defaults set here : https://github.com/debops/debops/blob/master/ansible/roles/debops-contrib.dropbear_initramfs/defaults/main.yml#L31
are correct

But the logic to determine the authorized_keys location, based on package name, is wrong here :
https://github.com/debops/debops/blob/master/ansible/roles/debops-contrib.dropbear_initramfs/tasks/main.yml#L43
because on this system/dropbear-initramfs version, it is expected to deploy the authorized_keys file in ```/etc/initramfs-tools/root/.ssh```
